### ISSUE TITLE
User research

### ISSUE THREAD

**Issue by oliviertassinari** [634750802/ossinsight-next#58]
I'm coming from https://ossinsight.io/ which is built on Material UI. We are using OSS Insight to create custom dashboards to monitor the work of the community on our open-source repositories: https://tools-public.mui.com/prod/pages/9r8fshsf. I have noticed that the next version will be based on Tailwind CSS + Radix, so dropping Material UI. 

This feels like a great product feedback opportunity. I would love to learn more about the thought process for the change. What's the appeal? What made [Base UI](https://mui.com/base-ui/) a nonviable option? Thanks!

**Comment by 634750802** [comment]
Hi, @oliviertassinari, we love Material UI, it's wonderful. 

#### Why Tailwind + Radix

The major reason was we have decided to use Next13 app dir and the RSC was not quite compatible with lots of libraries at that time. So I just chose a easy way (Tailwind + Radix) to get things start.


#### Why dropped Material UI

- The site is not following material design and it's a bit hard to un-material MUI.
- We need a `RSC component` and a `client component` to create a styled page or layout in Next 13 App dir. It

**Comment by 634750802** [comment]
Hi @oliviertassinari , we all love the wonderful MUI, but we have several concerns about UI libraries in this project:
- The Next13 app dir and RSC are nice but not easy to use at this time. So I have chosen Tailwind CSS for styling which is more easy to integrate with NextJS compared to other CSS-in-JS solutions.
- The OSSInsight is not using material design and it's a bit hard to un-material MUI.

The new project is still in early version, we are focusing on feature not UI library for now. And the decision is not final, we will try Base UI in the future.

Thanks for your watching ❤️

**Comment by danilo-leal** [comment]
@634750802 thanks for the answer; really appreciate it! Out of curiosity, were you aware that Base UI or even [Joy UI](mui.com/joy-ui/getting-started/) (these two being alternative UI libraries developed by MUI) existed before Olivier's comment? 😬

**Comment by 634750802** [comment]
@danilo-leal Yes, I found them on document site before. They are very suitable for MUI developers.

For this project, Joy UI is in same situation as MUI and Base UI is lack of some components. Both of them are in beta version now so I didn't try.

But I would like to try BaseUI + Tailwind CSS in my personal projects!🚀 🚀 ----- without next13 appdir 🤣

**Comment by oliviertassinari** [comment]
@634750802 Love it, thanks! So I hear:

1. Next.js integration, we should all-in on https://github.com/mui/material-ui/issues/34905
1. Removing style, we should all-in on a CLI to turn Material UI to Base UI + styles: https://github.com/mui/material-ui/issues/38214.
